packages/chakra-admin/src/components/forms/BaseForm.tsx

```diff
@@ -109,14 +111,48 @@ export const BaseForm: FC<BaseFormProps> = ({
           }
         })}
       </chakra.div>
-      <chakra.div>
-        <SubmitButton
-          disabled={mutationResult?.loading}
-          isLoading={mutationResult?.loading}
-          type="submit"
-        />
-        <CancelButton />
-      </chakra.div>
+      {renderingInModal ? (
+        <DrawerFooter
+          zIndex="10"
+          position="absolute"
+          bottom="0px"
+          w="100%"
+          right="0px"
+          boxShadow="main"
+          bg="white"
+          p={4}
+        >
+          <chakra.div style={{ display: 'flex', justifyContent: 'end' }}>
+            <CancelButton />
+            <SubmitButton
+              disabled={mutationResult?.loading}
+              isLoading={mutationResult?.loading}
+              type="submit"
+              bg="gray.700"
+              _hover={{ bg: 'gray.500' }}
+              ml={5}
+            />
+          </chakra.div>
+        </DrawerFooter>
+      ) : (
+        <chakra.div
+          style={{
+            display: 'flex',
+            justifyContent: 'end',
+            marginBottom: '20px',
+          }}
+        >
+          <CancelButton />
+          <SubmitButton
```

Comment:
Can we find a way to not render two different versions of the button?
Just the container should change, and not also the children.